internal/api/router/router.go

```diff
@@ -0,0 +1,19 @@
+package router
+
+import (
+	"net/http"
+
+	"github.com/GoogleCloudPlatform/gcs-metadata-server/internal/api/handler"
+	"github.com/GoogleCloudPlatform/gcs-metadata-server/internal/repo"
+)
+
+func New(db *repo.Database) *http.ServeMux {
+	mux := http.NewServeMux()
+
+	exploreRepo := repo.NewExploreRepository(db)
+	exploreHandler := handler.NewExploreHandler(exploreRepo)
+
+	mux.HandleFunc("GET /explore/{path...}", exploreHandler.HandleExplore)
+
+	return mux
+}
```

Comment:
future improvement: I'm not sure that this small logic needs a standalone package, consider moving this back to cmd in the future.

Reply:
Sounds good! I want to see how much space does this init function occupy for now, but otherwise I will refactor accordingly